### ISSUE TITLE
The multi sort on the table component is not keyboard accesible.

### ISSUE THREAD

**Issue by jpfaehler** [primefaces/primeng#19559]
### Describe the bug

The multi sort is not keyboard accessible. When tabbed to a column you can hit enter to sort just that column, which resets all other column sorting. But if you hold down the metaKey and try to hit enter it will not multi-sort as expected. This is unfortunately a WCAG AA compliant issue, specifically for chapter 2.1.1.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/run?file=src%2Fapp%2Ftable-multiplecolumnssort-demo.ts

### Environment

This is broken on your demo site.

### Angular version

21

### PrimeNG version

v21

### Steps to reproduce the behavior

1. Tab to a column with multi sort enabled.
2. Try to hold down the metaKey and press enter
3. Notice that it doesn't add to the multi sort

### Expected behavior

It adds to the multi sort.
